### PR TITLE
feat(native): support list/describe/ssh command for native provider

### PR DESCRIPTION
**Pull request by JacieChao**
https://github.com/cnrancher/autok3s/issues/330

### CODE REVIEW

File: pkg/server/store/websocket/log.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func WriteLastLogs(t *tail.Tail, w http.ResponseWriter, f http.Flusher, logFileP
 // nolint: gocyclo
 func logHandler(apiOp *types.APIRequest) error {
 	cluster := apiOp.Request.URL.Query().Get("cluster")
-	provider := apiOp.Request.URL.Query().Get("provider")
+	//provider := apiOp.Request.URL.Query().Get("provider")
```
Comment (Jason-ZW) [inline pkg/server/store/websocket/log.go:75]: remove useless line

File: pkg/providers/native/flag.go [hunk 2]
```diff
@@ -14,18 +18,31 @@ const createUsageExample = `  autok3s -d create \
 const joinUsageExample = `  autok3s -d join \
     --provider native \
     --ssh-key-path <ssh-key-path> \
-    --ip <existing server ip> \
```
Comment (Jason-ZW) [inline pkg/providers/native/flag.go:17]: why remove `--ip` flag?  Are you going to use '-master-ips' to try to join an existing cluster ？

**Review by Jason-ZW — CHANGES_REQUESTED** [review]

File: pkg/providers/native/flag.go (hunk 2 above)
Comment (JacieChao) [reply pkg/providers/native/flag.go:17]: We have saved the native cluster to DB, so we can get `--ip` value from DB. User is no need to specify this parameter when trying to join nodes to this cluster.

Comment (Jason-ZW) [reply pkg/providers/native/flag.go:17]: If so, is the `--name` attribute missing?

Comment (JacieChao) [reply pkg/providers/native/flag.go:17]: yes, I will fix it.

**Review by Jason-ZW — APPROVED** [review]

**Pull request merged**